modules/cli/src/main/scala/scala/cli/launcher/LauncherCli.scala

```diff
@@ -70,6 +70,15 @@ object LauncherCli {
     sys.exit(exitCode)
   }
 
+  def scalaCliVersion(cliVersion: String, cliScalaVersion: Option[String]): String =
```

Comment:
Maybe this can be renamed to either `scalaVersion` or `scalaCliScalaVersion`? `scalaCliVersion` is confusing IMO (at first, I though it was the version of Scala CLI itself somehow).